diff --git a/node_modules/react-native-google-mobile-ads/android/app-json.gradle b/node_modules/react-native-google-mobile-ads/android/app-json.gradle
index 3a687fd..cf5a4af 100644
--- a/node_modules/react-native-google-mobile-ads/android/app-json.gradle
+++ b/node_modules/react-native-google-mobile-ads/android/app-json.gradle
@@ -50,7 +50,7 @@ if (jsonFile != null && jsonFile.exists()) {
       }
     }
   } else {
-    rootProject.ext.googleAdsJson = false
+    rootProject.ext.googleMobileAdsJson = false
     rootProject.logger.info ":${project.name} ${fileName} found with no ${jsonRoot} config, skipping"
     android {
       defaultConfig {
